### DISCUSSION TITLE
Horizon Slack notifications

### DISCUSSION THREAD

**Discussion by siarheipashkevich** [laravel/framework#51879] [Q&A]
How to set the Horizon Slack notifications (using channel not webhook)
https://laravel.com/docs/11.x/horizon#notifications

When using new Slack API:
https://laravel.com/docs/11.x/notifications#slack-notifications

Can I just add `SLACK_BOT_USER_OAUTH_TOKEN` and set `Horizon::routeSlackNotificationsTo('#horizon');` ?

**Comment by wizzymore** [comment]
I don't think you can. Currently from reading through the horizon source code it still uses the old webhook implementation, both for the logic of sending and the way the message is built.
